projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -266,6 +268,20 @@ public function initial_state() {
 		return $state;
 	}
 
+	public function get_services() {
+		$site_id = Connection_Manager::get_site_id();
```

Comment:
This API call should probably be cached, so we're not making another API request on each load of the admin page, especially as this data will not change very often